docs/operational_guide/repairs.md

```diff
@@ -0,0 +1,34 @@
+# Background Repairs (beta)
+
+## Overview
+
+An M3DB cluster can be configured to passively repair itself in the background. If the background repairs feature is enabled, then M3DB nodes will passively scan each other's metadata and when a mismatch is detected the nodes will perform a repair such that each node in the cluster eventually settles on a consistent view of the data.
```

Comment:
> If the background repairs feature is enabled, then M3DB [...]

"If background repairs are enabled, M3DB [...]"

> each other's metadata and when a mismatch is detected the nodes will [...]

"the metadata of other nodes. If a mismatch is detected, affected nodes will [...]

also, it's probably worth clarifying (a) which nodes are actually doing the repairing and (b) how those nodes are selected (i.e., if nodes A and B have a mismatch, is it recency, is it quorum with other nodes, is it a coin flip, etc).

Reply:
Good point, I updated it to explain how the repairs work. TLDR there is no coordination, each node detects mismatches on its own and then repairs itself by streaming the data from its peers and merging all the data together into one new stream (which basically works cause M3DB streams are basically CRDTs except for the one where there are different values for the same timestamp which can cause a bit of flip-flopping back and forth)